openml/_api_calls.py

```diff
@@ -42,7 +43,7 @@ def _perform_api_call(call, request_method, data=None, file_elements=None):
     url += call
 
     url = url.replace('=', '%3d')
-
+    logging.info('[%s] %s' % (request_method, url))
```

Comment:
Hm, I think it would make a lot of sense if you also add a `logging.info` for when the call is done, which then also states how much time was spent in that function.